### DISCUSSION TITLE
note/node-api-develop-summary/

### DISCUSSION THREAD

**Discussion by giscus** [gitguanqi/gitguanqi.github.io#9] [Announcements]
# note/node-api-develop-summary/

导语：之前做过一个小项目，其中后端用的node开发的接口服务，有些感觉挺好的方案，希望可以总结归纳一下，以免日后忘记，留个备份，记录一下。
目录
简介
规范
工具
案例
简介
定义
说到接口，大家应该都不陌生，对于程序员朋友来说是很常见的一种东西。API全称是Application Programming Interfac ...

http://localhost:8080/note/node-api-develop-summary/

**Comment by gitguanqi** [comment]
写的真好
